save_today.py

```diff
@@ -1,54 +1,72 @@
 #!/usr/bin/env python
 
 import argparse
-from datetime import date
+from datetime import date as dt
 
 from harris_county_bookings import *
 
 
 # noinspection PyUnusedLocal
 def lambda_handler(event, context):
-    save_to_github(ALL_MODES)
-    save_to_dataworld(ALL_MODES)
+    today = dt.today()
+    data = JIMSRecorder.fetch_and_clean_data(s3=True, date=today)
```

Comment:
Instead of parsing the raw data each time a new publisher is called, we'll just run it once and pass the result to the publishers.